src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
+@import "./variables";
+
 body {
+  display: flex;
+  justify-content: center;
+  font-size: 30px;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: $gap-size;
+  width: $ceil-width * 7 + $gap-size * 6 + 20px;
+  padding: 20px 10px 10px;
+
+  &__cell {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid black;
+    width: $ceil-width;
+    height: $ceit-height;
+    font-size: 30px;
```

Comment:
You already have `font-size` in the body - why duplicate it here?

In addition: per task:
> Add a number (Arial 30px)